### ISSUE TITLE
Add description to library and asset detail pages

### ISSUE THREAD

**Issue by mattrosno** [carbon-design-system/carbon-platform#558]
The library schema has a description. That needs rendered above the dashboard on the library detail page.

- [x] Link to redline @kingtraceyj
- [x] Open issue so @aubrey-oneal can update existing library descriptions https://github.com/carbon-design-system/carbon-platform/issues/559

For visual reference, this is adding A.
<img width="718" alt="image" src="https://user-images.githubusercontent.com/1691245/163266970-bf1e68c3-9c00-4c17-a101-763c873f8900.png">

**Comment by mattrosno** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Development-624c3fc8b7972e001130fef7/issues/carbon-design-system/carbon-platform/558?planning-poker) with ZenHub @alisonjoseph @francinelucca

**Comment by kingtraceyj** [comment]
library redlines: https://ibm.box.com/s/s6btwm637udnek3pbjtfem412kgw83hp

**Comment by mattrosno** [comment]
@andreancardona I set sprint 16.

**Comment by andreancardona** [comment]
@mattrosno sounds good thank you!

**Comment by francinelucca** [comment]
is there a separate issue to update lib descriptions once #559 is complete? or is #559 blocking this one? @mattrosno

**Comment by mattrosno** [comment]
@francinelucca since we're in a beta still (kindof a private beta as you need IBM employee auth to view), I don't believe #559 to be a blocker. But definitely something we'll want to do prior to v1.
